src/test/app/ValidatorSite_test.cpp

```diff
@@ -197,8 +197,12 @@ class ValidatorSite_test : public beast::unit_test::suite
             emptyLocalKey, emptyCfgKeys, cfgPublishers));
 
         using namespace std::chrono_literals;
+        // Normally, tests will almost never need all this time,
+        // but sometimes DNS resolution takes an inordinate amount
+        // of time. This appears to be an OS-level issue, so the
+        // test will just wait.
```

Comment:
I'd probably leave out the "this appears to be an OS-level issue". We never really dug into this, we just know that it take a long time.

Reply:
Good point. Will change.